libs/surfaces/console1/console1.cc

```diff
@@ -709,6 +749,18 @@ Console1::set_current_stripable (std::shared_ptr<Stripable> r)
 			}
 		}
 
+#ifdef mixbus
```

Comment:
Another `#ifdef`  should be `#ifdef MIXBUS`

Reply:
s.o.